core-bundle/contao/templates/_new/content_element/list.html.twig

```diff
@@ -6,5 +6,9 @@
 {% endblock %}
 
 {% block list_item %}
-    {%- if as_editor_view %}{{ item|raw }}{% else %}{{ item|insert_tag|raw }}{% endif -%}
+    {%- if as_editor_view %}
+        {{ item|raw }}
+    {% else %}
+        {{ item|insert_tag|raw }}
+    {% endif -%}
```

Comment:
```suggestion
    {%- if as_editor_view -%}
        {{ item|raw }}
    {%- else -%}
        {{ item|insert_tag|raw }}
    {%- endif -%}
```

Reply:
Changed in 8bd552f5952b80a01c9ef721233a31191f98e8f0.